devops/resources/create-data-czar.sh

```diff
@@ -9,17 +9,18 @@ create_virtualenv --python=python3.8 --clear
 set -u
 
 cd "$WORKSPACE/configuration"
-pip install -r requirements.txt
+pip install -r util/jenkins/requirements.txt
```

Comment:
Do you know if we use this requirements file for all Jenkins jobs?

Reply:
Not in all jenkins jobs, I have seem a few that are using main requirements.txt file. This was a suggestion from Nadeem in a review meeting we had on meet.